mbrl/diagnostics/training_browser.py

```diff
@@ -50,10 +51,9 @@ def reloadExperiments(self, data):
         self.data = []
         for path in data:
             if path.endswith(SOURCE):
+                config_path = Path(path).parent / Path(".hydra/config.yaml")
```

Comment:
nit: I think you don't need to also wrap ".hydra/config.yaml" into a `Path` object.